org.mixedrealitytoolkit.uxcomponents/Editor/CreateElementMenus.cs

```diff
@@ -210,6 +214,19 @@ private static void CreateActionButton(MenuCommand menuCommand)
             CreateElementFromPath(ActionButtonPath, menuCommand);
         }
 
+        [MenuItem("GameObject/UI/MRTK/Simple Button")]
+        private static void CreateSimpleButton(MenuCommand menuCommand)
+        {
+            GameObject simpleButton = CreateElementFromPath(SimpleButtonPath, menuCommand);
+            GameObject simpleButtonContentGameObject = simpleButton.transform.GetChild(3).gameObject;
```

Comment:
Please add the `Undo.RecordObject` operations, so callers can undo actions on components.

Follow pattern of `CreateActionButtonWide`

Reply:
Added them in latest commit.